src/style.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 30px 15px 15px;
+  width: 200px;
+}
+
+.image-wrap {
+  height: 134px;
```

Comment:
Since you set height explicitly to this block you have to center the image vertically and horizontally. Try to figure it out with `flexbox`